docs/Contribute/ContributeWrite/WriteStack.md

```diff
@@ -0,0 +1,63 @@
+# Contribute to the Software Stack
+
+:::warning
+Writing on the *software stack* makes changes directly to the BTCPay Server software that everyone uses. **Do not** change the code if you aren't familiar with it.
+:::
+
+::: tip
+If you have trouble finding a string or contributing to the software stack, ask the [community](../../Community.md)
+:::
+
+## **Step 1**
+
+Fork/Clone the main repository ([BTCPay Server](https://github.com/btcpayserver/btcpayserver/)) using Github Desktop and publish it.
+
+![ContributeCloneRepo](../../img/Contribute/ContributeCloneRepo.jpg)
+
+![ContributeCloneRepo2](../../img/Contribute/ContributeCloneRepo2.jpg)
+
+## **Step 2**
+
+Create a branch and name it (for example what file you're working on).
+
+![ContributeCreateBranch](../../img/Contribute/ContributeCreateBranch.jpg)
+
+![ContributeCreateBranch2](../../img/Contribute/ContributeCreateBranch2.jpg)
+
+## **Step 3**
+
+Now open your branch in your file explorer.
+
+![ContributeFindFile](../../img/Contribute/ContributeFindFile.jpg)
+
+You're all set ! 
```

Comment:
Why the spaces before the exclamations? I have never see punctuation like this.

Reply:
I exclusively use exclamations points like that, because in French it's done that way so some might slip in when writing in English =)

```suggestion
You're all set! 
```